spec/models/queen_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+RSpec.describe Queen, type: :model do
+  describe "#valid_move?" do
+    let(:game) {create :game}
+
+    it "should properly evaluate malformed moves that would otherwise be valid" do
+      white_queen = game.pieces.find_by(rank: 1, file: 1)
+      white_queen2 = game.pieces.find_by(rank: 1, file: 8)
+      white_queen3 = game.pieces.find_by(rank: 1, file: 3)
+      black_queen = game.pieces.find_by(rank: 8, file: 1)
+      black_queen2 = game.pieces.find_by(rank: 8, file: 8)
+      black_queen3 = game.pieces.find_by(rank: 8, file: 3)
```

Comment:
The pieces that you're pulling up here via `find_by` won't actually be queens, which means that you're testing other copies of `valid_move?`, rather than `Queen#valid_move?`